spec/index.html

```diff
@@ -132,73 +155,106 @@ <h3>Namespaces</h3>
+        <p>The simplest <a>N3 statement</a> or <a>triple</a> is a sequence of
+          <a data-cite="RDF11-CONCEPTS#dfn-subject">subject</a>, 
+          <a data-cite="RDF11-CONCEPTS#dfn-predicate">predicate</a>, and
+          <a data-cite="RDF11-CONCEPTS#dfn-object">object</a>,
+          separated by whitespace and terminated by '`.`'
+          after each <a>triple</a>.
+          For now, we will asssume that a resource is represented either by an <a href="#iris">IRI</a> or a <a href="#literals">literal</a>.
+          In general, one uses an IRI to identify, well, an identifiable entity,
+          such as a person, place, or thing &mdash;
+          a literal is used for a textual or numerical (i.e., datatype) value,
+          such as names, dates, height, and so on.</p>
+        <p>The three <a>N3 triples</a> below highlights the emnity between Spiderman and the Green Goblin, and lists the names of these two individuals:</p>
         <!-- 
           NOTE: there is a data-include attribute 
           (https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide#inclusions--transformations)
           that allows you to directly include a data file here
           -->
-        <pre class="example nohighlight">
-          &lt;http://example.org/#spiderman&gt; &lt;http://example.org/#enemyOf&gt; &lt;http://example.org/#green-goblin&gt; .
-          &lt;http://example.org/#spiderman&gt; &lt;http://xmlns.com/foaf/0.1/name&gt; "Spiderman" .
-          &lt;http://example.org/#green-goblin&gt; &lt;http://xmlns.com/foaf/0.1/name&gt; "Green Goblin" .
+        <pre class="example nohighlight" data-transform="updateExample">
+          <!--
+          <http://example.org/#spiderman> <http://example.org/#enemyOf> <http://example.org/#green-goblin> .
+          <http://example.org/#spiderman> <http://xmlns.com/foaf/0.1/name> "Spiderman" .
+          <http://example.org/#green-goblin> <http://xmlns.com/foaf/0.1/name> "Green Goblin" .
+            -->
         </pre>
-        <p class="note">There is no inherent order for N3 / RDF triples. This is the same for the relational model, i.e., relational tuples or rows do not have an inherent order. Hence, it is a mistake to associate meaning with the order of a statement in an N3 document: e.g., assuming that the first listed telephone for Spiderman would his landline, and the second one his mobile number.</p>
+        <p class="note">There is no inherent order for <abbr title="Notation3">N3</abbr> / RDF triples. This is the same for the relational model, i.e., relational tuples or rows do not have an inherent order. Hence, it is a mistake to associate meaning with the order of a statement in an <abbr title="Notation3">N3</abbr> document: e.g., assuming that the first listed telephone for Spiderman would his landline, and the second one his mobile number.</p>
```

Comment:
```suggestion
        <p class="note">There is no inherent order for <abbr title="Notation3">N3</abbr> or RDF triples. The same is true for the relational model, i.e., relational tuples or rows do not have an inherent order. Hence, it is a mistake to associate meaning with the order of statements in an <abbr title="Notation3">N3</abbr> document; for instance, it would be an error to assume that the first listed telephone for Spiderman would be his landline, and the second one his mobile number.</p>
```